modules/rational-defaults.el

```diff
@@ -39,15 +69,15 @@
 (customize-set-variable 'kill-do-not-save-duplicates t)
 
 ;; Make scrolling less stuttered
-(setq auto-window-vscroll nil)
+(customize-set-variable auto-window-vscroll nil)
```

Comment:
`auto-window-vscroll` is not a `defcustom` variable, thus `setq` is probably more appropriate here.